docs/user/features/note-templates.md

```diff
@@ -25,27 +25,40 @@ To create a note from a template:
 _Theme: Ayu Light_
 
 ## Special templates
+
 ### Default template
 
 The `.foam/templates/new-note.md` template is special in that it is the template that will be used by the `Foam: Create New Note` command.
-Customize this template to contain content that you want included every time you create a note.
+Customize this template to contain content that you want included every time you create a note. At minimum this template should include the following:
+
+```markdown
+---
+type: basic-note
+---
+```
 
 ### Default daily note template
 
 The `.foam/templates/daily-note.md` template is special in that it is the template that will be used when creating daily notes (e.g. by using `Foam: Open Daily Note`).
-Customize this template to contain content that you want included every time you create a daily note.
+Customize this template to contain content that you want included every time you create a daily note. At minimum this template should include the following details:
```

Comment:
I would put this more as a recommendation than a mandatory action

Reply:
Agreed, had the same issue as with the new-note template mentioned above. Will change it to read as more of a recommendation than a requirement.